bigquery/indexer.py

```diff
@@ -240,13 +243,56 @@ def read_table(client, table_name):
         client.dataset(dataset_id, project=project_id).table(table_name))
 
 
+def create_samples_json_export_file(es, index_name, project_id):
```

Comment:
Looks like the file has 1 line with 20918223 characters. Can you split into multiple lines, maybe one line per sample?

Reply:
Done--formatted `json.dumps()`.